build.gradle.kts

```diff
@@ -43,8 +51,13 @@ intellij {
 
 tasks {
     withType<org.jetbrains.intellij.tasks.BuildSearchableOptionsTask>()
-        .forEach { it.enabled = false }
+            .forEach { it.enabled = false }
     runIde {
         maxHeapSize = "1g"
     }
-}
\ No newline at end of file
+    // Test stuffs part 2
```

Comment:
I would change this comment or just delete it entirely

Reply:
Fixed with new commits